app/controllers/concerns/phone_number_localization.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module PhoneNumberLocalization
+  extend ActiveSupport::Concern
+
+  included do
+    include PatientPhoneNumberHelper
+
+    def localized_phone_number
+      number_with_country_code(phone_number)
```

Comment:
- Lets make this a model concern instead, maybe `PhoneNumberLocalizable` to go with the naming convention?
- Can move the body of `number_with_country_code` into `localized_phone_number`, and get rid of the `PatientPhoneNumberHelper` altogether.

Reply:
okay